flake.lock

```diff
@@ -0,0 +1,106 @@
+{
```

Comment:
Is putting a "lock" file under version control intentional?

Reply:
It is intentional. The point is to capture the SHA hashes of the dependencies for reproducibility purposes.